### PR TITLE
Pull request gustavo.arana6562

### PR DESCRIPTION
**Pull request by gustavo-arana**
# Steps to resolve exercise 1

1. Prepare your GitHub account:
    * Create or sign up for an account on [GitHub](https://github.com/)
    > If this link doesn´t work, copy and paste this URL into your browser ****https://github.com/****

    * Open this [link](https://gitgub.com/undefined-academy/semana-1) containing the repository to fork: Repository **semana-1**
    > If this link doesn´t work, copy and paste this URL into your browser ****https://gitgub.com/undefined-academy/semana-1****
    
2. On the **semana-1** repository, click on the **Fork** button and select **Create a new fork**
    > This action creates a copy of the **semana-1** repository in your GitHub account.

3. Open your terminal and execute the command `git clone [URL_OF_YOUR_FORKED_REPOSITORY]`
    * Open your forked repository on a browser
    * Click the **Code** button
    * Click the **SSH** tab
    * Copy the URL of your repository
    * Execute the command `git clone [URL_OF_YOUR_FORKED_REPOSITORY]` in your console
    > For example, `git clone git@github.com:gustavo-arana/semana-1.git`

4. Change to the folder of the repository with the command `cd semana-1`

5. Create a directory in the **github-profiles** a folder with the command `mkdir NAME_DIRECTORY`
    > The **NAME_DIRECTORY** will be your nickname on Discord and your account number: **gustavo.arana-6562**

6. Create a file named **README.md** with the command `touch README.md`

7. Commit the changes:
    * `git pull origin main` -> Check the changes on the remote repository
    * `git add .` -> Add new files
    * `git commit -m "Add README.md gustavo.arana-6562"` -> Commit changes
    * `git push origin main` -> Send changes to the remote repository

8. On your remote repository on the browser, click on the **Pull request** tab
    > You will see a **Comparing changes** section and the option to **Create pull request** to the base repository: undefined-academy/semana-1.

### CODE REVIEW

**Review by Oiranca — APPROVED** [review]
Me ha gustado, aunque el comando touch pondría touch <options> <file or directory name>

**Review by Oiranca — APPROVED** [review]
Me a gustado, sencillo y claro!! :clap:

**Comment by gustavo-arana** [comment]
> Me ha gustado, aunque el comando touch pondría touch

Hola Orianca, podrías por favor hacer el comentario en la linea especifica donde lo pondrías y el ejemplo.

File: github-profiles/gustavo.arana-6562/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Genexus is a great tool. **I used to say "why learn native if we have Genexus?"**, but I was wrong. However, it's never too late to learn HTML, CSS, and JavaScript, and that's why I'm here.
+
+## Most used commands in Linux
+
+| Command | Description                    | Example          |
+| ------- | ------------------------------ | ---------------- |
+| cd      | Change of directory            | cd home          |
+| mkdir   | Create a directory             | mkdir MyFolder   |
+| mv      | Move a directory               | mkdir MyFolder   |
+| touch   | Create a file                  | touch index.html |
```
Comment (Oiranca) [inline github-profiles/gustavo.arana-6562/README.md:16]: Perdona me refería esta línea. Un saludo

**Review by fjquants — APPROVED** [review]
Hola me ha parecido bien realizado el ejercicio

**Pull request merged**